src/bluesky/tests/test_seq_num_indices_in_sync.py

```diff
@@ -124,8 +124,7 @@ def collect_stream_datum_repeat_stream_resource(self) -> Iterator[StreamAsset]:
             # New stream_resource half way through the run
             if DRAFT_0_STREAM_RESOURCE:
                 self.stream_resource_compose_datum_pairs = tuple(
-                    self.compose_stream_resource("", "", f"non_existent_{det}.hdf5", det, {})
-                    for det in self.detectors
+                    self.compose_stream_resource("", f"non_existent_{det}.hdf5", det, {}) for det in self.detectors
```

Comment:
As above, though arguably since this is a test we can always assume the bundled version of event-model. We should take these out in a different PR if that's the case though.